deliverables/README.md

```diff
@@ -31,38 +37,98 @@ What is the role of Javascript in a web page?
 
 #### meta
 Why do we separate HTML and CSS?  
+We separate the two so that the pages aren't overly cluttered when editing code. We can easily move from one to the other and edit them easier than if we were to put all of the CSS in-line with the HML.
 
 #### HTML
 What are classes and IDs (and how are they different)?  
+Classes can be used on multiple elements while IDs must contain unique values. IDs are declared in CSS documents using the "#" before the ID name while classes are defined with a "."
+
 What are elements?  
+Elements are what are found in between an opening and closing tag.
+
 What are tags?  
+Tags surround elements and give them meaning.
+
 What are attributes?  
+Attributes are extra information, usually found within an opening tag.
+
 What are forms?  
+Forms are used to collect input from users. From then, the form can manipulate the data according to how it's coded.
+
 What is a div?  
+Divs are placeholders in which users can add properties of CSS to chunks of information.
+
+What are selectors?
+Selectors are the CSS equivalents of tags, which surround properties to give them meaning. 
```

Comment:
Selectors are similar to tags, but what they do is allow our CSS to 'select' particular HTML elements. They can even be tags, if you wanted to apply some particular styling to all your HTML tags of that type, e.g., `p {color: blue;}` would make all paragraph tag content blue.